citar.el

```diff
@@ -8,7 +8,7 @@
 ;; License: GPL-3.0-or-later
 ;; Version: 0.8
 ;; Homepage: https://github.com/bdarcus/citar
-;; Package-Requires: ((emacs "26.3") (bibtex-completion "1.0") (parsebib "3.0"))
+;; Package-Requires: ((emacs "27.2") (s "1.12") (parsebib "3.0"))
```

Comment:
Is there a particular reason to require 27.2? On Debian stable there is still only 27.1 available, which I use. And it works reasonably well. I consider dropping support for 26 to be justified.